praetorian_cli/ui/console/commands/tools.py

```
@@ -543,3 +543,172 @@ def _cmd_capabilities(self, args):
+                    self.console.print(f'[dim]{tool_name}: not installed[/dim]')
+                continue
+
+            mod = reg.get_module(tool_name)
+            if not mod:
+                continue
+
+            current = reg.get_version(tool_name)
+            current_ver = current['version'] if current else 'unknown'
+
+            try:
+                ver_result = subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)